python/mxnet/torch.py

```diff
@@ -0,0 +1,157 @@
+# coding: utf-8
+"""Interface for NDArray functions executed by torch backend.
+Install torch and Compile with USE_TORCH=1 to use this module"""
+from __future__ import absolute_import
+
+import ctypes
+import sys
+from .base import _LIB
+from .base import c_array, py_str, ctypes2docstring
+from .base import mx_uint, mx_float, NDArrayHandle, FunctionHandle
+from .base import check_call
```

Comment:
Hmm, is it possible to simply not override, but expose them as different names? The idea is to have torch as a complement of what we have here